scripts/cybershake/queue_monitor.py

```diff
@@ -151,6 +153,49 @@ def update_tasks(
                     "Disappeared from squeue. Creating a new task.",
                 )
             )
+            # When job failed, we want to log metadata as well
```

Comment:
Extract the logging metadata into a separate function as we will eventually put this logic into completed cases too.